src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/StatisticalTableFunction.cs

```diff
@@ -69,38 +71,60 @@ public override bool IsServerDelegatable(CallNode callNode, TexlBinding binding)
 
                 return false;
-            }
+            }
+
+            var args = callNode.Args.Children.VerifyValue();
+            var fieldType = binding.GetType(args[1]);
+
+            // if a non-numeric feature is enabled and the type is not a number, also check for the specific sub type capability
+            if (binding.Document.Properties.EnabledFeatures.IsDateTimeMinMaxDelegationEnabled && fieldType != DType.Number && !TryGetValidDataSourceForNonNumericDelegation(callNode, binding, out var nonNumericDataSource))
+            {
+                SuggestDelegationHint(callNode, binding);
+                TrackingProvider.Instance.SetDelegationTrackerStatus(DelegationStatus.NotANumberArgType, callNode, binding, this, DelegationTelemetryInfo.CreateUnsupportArgTelemetryInfo(binding.GetType(args[1])));
+
+                return false;
+            }
 
-            var args = callNode.Args.Children.VerifyValue();
             if (binding.GetType(args[0]).HasExpandInfo
             || (!binding.IsFullRecordRowScopeAccess(args[1]) && args[1].Kind != NodeKind.FirstName)
             || !binding.IsRowScope(args[1])
-            || binding.GetType(args[1]) != DType.Number
+            || !SupportedTypes(binding).Contains(fieldType)
             || ExpressionContainsView(callNode, binding))
             {
                 SuggestDelegationHint(callNode, binding);
-
+ 
                 if (binding.GetType(args[1]) != DType.Number)
                 {
                     TrackingProvider.Instance.SetDelegationTrackerStatus(DelegationStatus.NotANumberArgType, callNode, binding, this, DelegationTelemetryInfo.CreateUnsupportArgTelemetryInfo(binding.GetType(args[1])));
                 }
                 else
                 {
                     TrackingProvider.Instance.SetDelegationTrackerStatus(DelegationStatus.InvalidArgType, callNode, binding, this, DelegationTelemetryInfo.CreateUnsupportArgTelemetryInfo(binding.GetType(args[1])));
-                }
+                }
 
                 return false;
             }
-
-            if (binding.IsFullRecordRowScopeAccess(args[1]))
-            {
-                return GetDottedNameNodeDelegationStrategy().IsValidDottedNameNode(args[1].AsDottedName(), binding, null, null);
-            }
-
-            var firstNameStrategy = GetFirstNameNodeDelegationStrategy().VerifyValue();
-            return firstNameStrategy.IsValidFirstNameNode(args[1].AsFirstName(), binding, null);
-        }
-
+
+            if (binding.IsFullRecordRowScopeAccess(args[1]))
+            {
+                return GetDottedNameNodeDelegationStrategy().IsValidDottedNameNode(args[1].AsDottedName(), binding, null, null);
+            }
+
+            var firstNameStrategy = GetFirstNameNodeDelegationStrategy().VerifyValue();
+            return firstNameStrategy.IsValidFirstNameNode(args[1].AsFirstName(), binding, null);
+        }
+
+        protected virtual IList<DType> SupportedTypes(TexlBinding binding)

```

Comment:
Consider: `bool IsSupportedTypeForDelegation(TexlBinding binding, DType)`

Might be better than having the `contains` check in the calling code?